public/robots.txt

```diff
@@ -3,3 +3,13 @@
 # To ban all spiders from the entire site uncomment the next two lines:
 # User-Agent: *
 # Disallow: /
+
+User-agent: *
+Disallow: /dbhero/
+Disallow: /api_token
+Disallow: */admin
+Disallow: */new-admin
+Disallow: */projects/
+Disallow: */users/*/edit
+
+Sitemap: https://www.catarse.me/sitemap.xml
```

Comment:
@begrossi this is the right path? if yes wee need to add a route on config/routes.rb